sequencer/src/block.rs

```diff
@@ -190,6 +202,19 @@ impl Header {
             transactions_root,
             fee_merkle_tree_root,
             block_merkle_tree_root,
+            builder_address: None,
+            builder_signature: None,
+            builder_fee_amount: None,
```

Comment:
Very important that the address and fee are set _before_ signing, so that an adversary can't change the builder or cause the builder to pay more than intended.